src/pages/Index.vue

```diff
@@ -343,9 +348,12 @@ export default {
             if (priority[0]) {
                 galaxies.splice(0, 0, priority[0]);
             }
-            return galaxies;
+            this.prioritizedGalaxyLocales = galaxies;
```

Comment:
Side effects in computeds need to be avoided

Reply:
Just gave this another look and it'd be cleaner to refactor this to be a computed without extra state set on mount, but it's not going to cause any immediate problems.

Reply:
@dannon thanks for the extra set of eyes on this! I previously had this as a computed on `master` in [this commit](https://github.com/galaxyproject/galaxy-hub/pull/2411/commits/54231fbc1060afd3299fe31979b2d7bf4a33738d#diff-75f76a7ed2a0f45f79ae96d58b3530c71c8b88cc3612a84ffd784ba8df2e3cbaL335-L346) but I was under the impression it was contributing to the labeling misalignment from the [gif video](https://github.com/galaxyproject/galaxy-hub/pull/2411#issue-2145760884) at the top of this PR so I removed it - my thinking was that using computed wasn't rehydrating the DOM with the updated compute values potentially because calling `getTimezoneOffset()` from one's browser was slower somehow on Production (the error wasn't fully reproducible on local). As long as the CI tests pass, I'm fine with merging without this change, but I'm still curious to know what you think the real culprit was for the misaligned-labeling issue? Perhaps my use of `[0]` referencing an Object from [this line](https://github.com/galaxyproject/galaxy-hub/commit/5ca4cecdb1d1fe46db6ec0759cc361d0a5e875de#diff-75f76a7ed2a0f45f79ae96d58b3530c71c8b88cc3612a84ffd784ba8df2e3cbaL18)..?

Reply:
@hujambo-dunia I'll check out the busted ref and see if I notice anything, I'll go ahead and merge this as-is now, though, and can make a refactoring pass after.